relay_timer_pool.go

```diff
@@ -51,9 +52,10 @@ func (rt *relayTimer) OnTimer() {
 	rt.pool.trigger(items, id, isOriginator)
 }
 
-func newRelayTimerPool(trigger relayTimerTrigger) *relayTimerPool {
+func newRelayTimerPool(trigger relayTimerTrigger, verifyPool bool) *relayTimerPool {
 	return &relayTimerPool{
-		trigger: trigger,
+		trigger:    trigger,
+		verifyPool: verifyPool,
```

Comment:
nit: since this is the pool struct, name might be nicer as just `verify` or `shouldVerify`.

Reply:
👍 Renamed to `verify`